scripts/create-component

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
```

Comment:
Can we stop writing bash scripts and write this in JS? Bonus points for a small test case 👍